lib/Transforms/IPO/StructFieldCacheAnalysis.cpp

```diff
@@ -385,18 +469,30 @@ static void performIRAnalysis(Module &M,
   DEBUG_WITH_TYPE(DEBUG_TYPE_IR, StructManager->debugPrintAnnotatedModule());
 }
 
+static void applyFilters(StructFieldAccessManager *StructManager) {
+  StructManager->summarizeFunctionCalls();
+  StructManager->applyFiltersToStructs();
+  StructManager->printStats();
```

Comment:
When to print what information is something I am confusing in my implementation now. I plan to print what I think it important on each branch and clean up the standard output in the final branch. Right now, the stats is kind of what I needed by default so I put it in the standard output. I think we can discuss what to put in the final output later.